billing-api/requirements.txt

```diff
@@ -1,43 +1,46 @@
-Babel==2.3.4
-bcrypt==3.1.1
-cffi==1.8.3
-click==6.6
-dateutils==0.6.6
-debtcollector==1.8.0
-docopt==0.6.2
-Flask==0.11.1
-Flask-Login==0.3.2
-funcsigs==1.0.2
-iso8601==0.1.11
-itsdangerous==0.24
-Jinja2==2.8
-keystoneauth1==2.12.1
-MarkupSafe==0.23
-mock==2.0.0
-monotonic==1.2
-msgpack-python==0.4.8
-MySQL-python==1.2.5
-netaddr==0.7.18
-netifaces==0.10.5
-oslo.config==3.17.0
-oslo.i18n==3.9.0
-oslo.serialization==2.13.0
-oslo.utils==3.16.0
-pbr==1.10.0
-positional==1.1.1
-pycparser==2.14
-pyparsing==2.1.9
-pytest==3.0.4
-python-dateutil==2.5.3
-python-keystoneclient==3.5.0
-pytz==2016.7
-records==0.4.3
```

Comment:
you removed this dependency yet you still require it in the usage_queries.py
This is what you get if you don't have this:

```bash
Traceback (most recent call last):
  File "/home/xxx/Documents/workspace/billing/billing-api/run.py", line 16, in <module>
    from billing_server.billing import app
  File "/home/xxx/Documents/workspace/billing/billing-api/billing_server/billing/__init__.py", line 28, in <module>
    from .usage_queries import Collaboratory
  File "/home/xxx/Documents/workspace/billing/billing-api/billing_server/billing/usage_queries.py", line 16, in <module>
    import records
ModuleNotFoundError: No module named 'records'
```